e2e/retro-pa/src/mspretro-contact.e2e-spec.ts

```diff
@@ -0,0 +1,36 @@
+import { browser, element, by } from 'protractor';
+import { ContactInfoPage } from './mspretro-pa.po';
+import { testPageLoad, testClickStepper, testClickContinue, testClickConsentModal, fillConsentModal } from '../../msp-generic-tests';
+import { FakeDataRetroPA } from './mspretro-pa.data';
+
+describe('MSP Retro PA - Contact Info Page', () => {
+    let contactInfoPage: ContactInfoPage;
+    const data = new FakeDataRetroPA;
+    let contactInfoData;
+    const SPOUSE_PAGE_URL = `msp/assistance/spouse`;
+    const CONTACT_PAGE_URL = `msp/assistance/contact`;
+    const REVIEW_PAGE_URL = `msp/assistance/review`;
+
+    beforeEach(() => {
+        contactInfoPage = new ContactInfoPage();
+        contactInfoData = data.contactInfo();
+        data.setSeed();
+    });
+
+    afterEach(() => {
+        browser.executeScript('window.sessionStorage.clear();');
```

Comment:
Idea - put these two calls into the MSPBaseTestPage. `page.clearStorage()` ?

Reply:
And this should be in beforeEach.  Optionaly it can be in both beforeEach and afterEach, but it must at least be in beforeEach.  It might seem overkill but it helps make sure our tests are reproducible. Otherwise we could have the same test fail because I have some bad local storage.